src/strongvelope/strongvelope.h

```diff
@@ -94,11 +94,6 @@ enum
 
 /** Contains a 64 bit user handle of a user that was excluded from chat */
     TLV_TYPE_EXC_PARTICIPANT    = 0x09,
-
-/** In case we encrypt keys using RSA, we can't reverse-decrypt them like with
- * EC, so we need to encrypt our own keys yo ourself so we can read our own
- * history messages. (legacy only) */
-    TLV_TYPE_OWN_KEY            = 0x0a,
```

Comment:
When you have enums that become obsolete/deprecated, it's better to leave traces in the code. In example, comment the line and add a comment-in-line: _deprecated (used for legacy encryption)_

Reply:
Otherwise, someone might consider as a typo and re-adjust the values, assigning `TLV_TYPE_INVITOR = 0x0a`.